scripts/bench_runner.py

```diff
@@ -0,0 +1,1111 @@
+
+    Args:
+        console: The rich console to print to.
+        alpha: The alpha value for statistical significance.
+        has_baseline: Whether a baseline benchmark was run.
+    """
+    console.print("Metric key:")
+
+    conf = int(100 * (1.0 - alpha))
+
+    name = "BenchmarkName..."
+    delta_icon = str(DeltaKind.IMPROVEMENT)
+    delta = "[faster]<delta>[/faster]"
+    p = "p=<U-test P-value>"
+    base_median = "[base_median]<median>[/base_median]"
+    base_conf = f"[base_conf]<% at {conf}th conf>[/base_conf]"
+    exp_median = "[exp_median]<median>[/exp_median]"
+    exp_conf = f"[exp_conf]<% at {conf}th conf>[/exp_conf]"
+
+    key_table = Table.grid(
+        Column(justify="right"),
+        Column(),
+        Column(),
+        Column(),
+        Column(),
+        padding=(0, 1),
+    )
+    if has_baseline:
+        key_table.add_row(name, delta_icon, delta, "", p)
+        key_table.add_row("baseline:", "", base_median, "±", base_conf)
+        key_table.add_row("experiment:", "", exp_median, "±", exp_conf)
+    else:
+        key_table.add_row(name, "", exp_median, "±", exp_conf)
+        # Only display comparable key if we have comparables to display.
+        if bool(comp_mapping.name_to_comp_tag):
+            key_table.add_row("vs Comparable:", delta_icon, delta, p)
+            key_table.add_row("", "", base_median, "±", base_conf)
+    console.print(Padding(key_table, (0, 0, 1, 3)))
+
+
+def print_results_table(
+    console: Console,
+    alpha: float,
+    has_baseline: bool,
+    metrics: list[str],
+    benchmark_names: list[str],
+    benchmark_metrics: dict[str, dict[str, BenchmarkRunMetrics]],
+    comp_mapping: ComparableBenchmarkMapping,
+) -> None:
+    """Builds and prints the main results table.
+
+    This function constructs a rich `Table` to display the benchmark results,
+    including deltas, medians, and confidence intervals for each metric. It thin
+    prints this to the provided `console`.
+
+    Args:
+        console: The rich console to print to.
+        metrics: A list of metric names to be displayed as columns.
+        benchmark_names: A list of main benchmark names for the rows.
+        alpha: The alpha value for statistical significance.
+        benchmark_metrics: A nested dictionary containing the collected metrics
+                           for each benchmark and metric.
+        has_baseline: Whether a baseline benchmark was run.
+        comp_mapping: The mapping of comparable benchmarks.
+    """
+    METRIC_TITLES = {
+        "real_time": "Wall Time",
+        "cpu_time": "CPU Time",
+        "iterations": "Iterations",
+    }
+
+    name_width = max(
+        (
+            len(name)
+            for name in (
+                benchmark_names
+                + [
+                    f"vs {tag}:"
+                    for tag in comp_mapping.name_to_comp_tag.values()
+                ]
+                + ["... experiment:"]
```

Comment:
Maybe explain this `... ` here?

Reply:
The `... ` here was a hold over from that being on the actual labels below. It isn't any more, so removed here.